### ISSUE TITLE
Browsing Itunes Doesn't Load Podcast

### ISSUE THREAD

**Issue by sammerry** [tkem/mopidy-podcast-itunes#8]
When using the mopidy web ui or ncmpcpp to browse `iTunes store: Podcasts` the tree will render until you reach a podcast and an error is thrown.
```
ERROR    Error retrieving podcast+https://acalltolead.libsyn.com/rss: 'RssFeed' object has no attribute 'expire'
```

If i copy/add the link above into a playlist several podcast tracks are added.
```
podcast+https://acalltolead.libsyn.com/rss
```

**Comment by tkem** [comment]
Sorry, cannott confirm. Browsing and playing "iTunes Store: Podcasts" -> "Business" -> "Careers" -> "A Call to Lead" works fine for me (using Mopidy-Podcast-iTunes 2.0.0 and Mopidy Mobile as a client). Other podcasts also seem to work fine when accessed through iTunes.

What's your Mopidy-Podcast-iTunes version?

**Comment by sammerry** [comment]
I'm running it in a debian docker container.

System
```
PRETTY_NAME="Debian GNU/Linux 8 (jessie)"
NAME="Debian GNU/Linux"
VERSION_ID="8"
VERSION="8 (jessie)"
```

Mopidy
```
Mopidy 2.2.1
Mopidy-Podcast==2.0.2
Mopidy-Podcast-iTunes==2.0.0
```

I can publish the docker container later tonight if you want access.

**Comment by tkem** [comment]
no, thanks, wouldn't know what to do with it anyway. the full stack trace given with the error message would probably be more helpful.
